decidim-design/app/views/decidim/design/components/cards/_static-card-g-highlight.html.erb

```diff
@@ -0,0 +1,20 @@
+<a class="card__highlight" id="participatory_process_highlight_1" href="/processes/corpse-curriculum">
```

Comment:
Having a link here would lead out the user from design app 
```suggestion
<a class="card__highlight" id="participatory_process_highlight_1" href="#">
```